### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by mikmaz**
- Fixed updater RBAC.
- Reduced checksum hash size.
- Removed duplicated webhook port definition.

### CODE REVIEW

File: config/admission-controller/admission_controller.yaml [hunk 1]
```diff
@@ -28,7 +28,4 @@ spec:
             requests:
               cpu: 20m
               memory: 20Mi
-          ports:
```
Comment (zimnx) [inline config/admission-controller/admission_controller.yaml:31]: why is this removed?

Comment (cezarybednarz) [reply config/admission-controller/admission_controller.yaml:31]: Because port for webhook is already defined in `admission_controller_webhook_patch.yaml` and it generated this error on deployment
```
validatingwebhookconfiguration.admissionregistration.k8s.io/scylla-operator-autoscaler-validating-webhook-configuration configured
The Deployment "scylla-operator-autoscaler-admission-controller" is invalid: spec.template.spec.containers[0].ports[1].name: Duplicate value: "webhook"
```

**Review by zimnx — APPROVED** [review]
